src/satosa/micro_services/ldap_attribute_store.py

```diff
@@ -443,7 +436,13 @@ def process(self, context, data):
         # and find asserted values to construct the ordered list of values for
         # the LDAP search filters.
         for candidate in config["ordered_identifier_candidates"]:
-            value = self._construct_filter_value(candidate, data)
+            value = self._construct_filter_value(
+                candidate,
+                data.subject_id,
+                data.subject_type,
+                data.auth_info.issuer,
+                data.attriutes,
```

Comment:
attributes instead of attriutes :upside_down_face:

Reply:
Your comment overlapped with me finding it and fixing it in my latest commit.